suzieq/sqobjects/basicobj.py

```diff
@@ -117,10 +119,15 @@ def _check_input_for_valid_args(self, good_arg_list, **kwargs,):
     def _check_input_for_valid_vals(self, good_arg_val_list, **kwargs):
         '''Check if the input is valid for the arg, if possible'''
 
-        if not good_arg_val_list:
-            return
-
+        fields = self.schema.fields
         for arg, val in kwargs.items():
+
+            # if we have integer fields, make sure we catch things like
+            # user specifying "> 25" and treating them as a single val
+            # instead of 2 separate args due to the split on space
+            if arg not in fields:
+                continue
```

Comment:
The comment seems unrelated to what happens here.

Reply:
Ack, removing.